src/main/java/gov/nasa/pds/tools/label/XMLCatalogResolver.java

```diff
@@ -349,6 +349,21 @@ public LSInput resolveResource(String type, String namespaceURI,
         // Ignore. Should not happen!!!!
       }
       
+	  // Similar to CachedLSResourceResolver, if systemId == null
+	  // we won't be able to resolve this resource
+      if (getProblemHandler() != null) {
```

Comment:
@jordanpadams Sorry for the confusion. This if block should go below within the:

````
if (systemId == null) {
  return null;
}
````
at line 364 just before returning null

Reply:
doh! 👎 for me. this is why I shouldn't be coding anymore!